licensing/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
+  - sig-release-leads
+  - justaugustus
+  - nikhita
+  - swinslow
```

Comment:
@swinslow Can you [apply](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E) to be a Kubernetes org member to have approval powers? :)

You can find more details [here](https://github.com/kubernetes/community/blob/master/community-membership.md).

Reply:
Will do -- thanks @nikhita  :)